### ISSUE TITLE
j.cue : recall a state with a custom ramp

### ISSUE THREAD

**Issue by avilleret** [jamoma/JamomaMax#980]
how can I recall a j.cue state with a custom value ?
for example I have a cue with some ramp time defined, I would like to go to that cue in 4500 ms from current state whatever its ramp value is
how can I override that ?

afaik using a [interpolate( message doesn't work since I want to interpolate from current state (which is not a cue)

**Comment by theod** [comment]
I add @jln, @reno-, @bltzr and @matcham to discuss about this because there are particularly involved into the design of j.cue.

IHMO j.cue should send what is written into the cue. For more advanced mechanism I would recommend to hack the content of the cue using the output message (or to use i-score :-)

**Comment by avilleret** [comment]
yes, I'm actually using output message to get cue content then, add ramp 1000 to the value
but would like to hear other users feedback on such a feature

**Comment by reno-** [comment]
I'm thinking about it for a few days, and I think that this may not be all managed in the cue-manager.
The default ramp might be a parameter attribute, which will be overwrite when you send a message to this parameter from anywhere.
Or maybe I'm a bit off topic, and a simplier(?) answer could be to have the ramp as an argument of the cue-play. So the ramp values won't be add into each lines of each cue, but magically/internally sent when playing cues. 
e.g. : recall init => play init cue
recall init ramp 1000 => send each value with ramp 1000
recall init ramp 4500 => send each value with ramp 4500

**Comment by theod** [comment]
@reno this sounds like a doable feature !
